### DISCUSSION TITLE
Add support for Ant Bms

### DISCUSSION THREAD

**Discussion by lucasimons** [syssi/esphome-jk-bms#38] [Ideas]
Hi,

I tried to search and group the various documentations but unfortunately I am not able to manage a new repository, given the excellent work with the bms jk could you help me with ant bms?

https://github.com/esphome/feature-requests/issues/1396

**Comment by syssi** [comment]
Do you have a working setup (ESP + BMS) already to try some basic UART communication?

**Reply by lucasimons** [reply]
Yes I have two bms, one of 100a already assembled and I have another one of 300a that I wanted to use for a second battery but due to the global situation I have not received the cells however I can use some smaller cells and do the tests with that, currently I use a raspberry connecting via bluetooth but I would like to switch to the cable connection via esphome, a part of the code should have already been written but it would be useful to create an extenal component on esphome

**Reply by syssi** [reply]
Cool. Let discuss all further steps here: https://github.com/syssi/esphome-ant-bms/issues/1
